setup.py

```diff
@@ -84,6 +84,7 @@
     extras_require=extras_require,
     setup_requires=setup_requires,
     tests_require=tests_require,
+    scripts=['scripts/build-manifests.sh'],
```

Comment:
Do we need to install this script? If it gets installed under `/some/prefix/bin/build-manifests.sh`, then we should perhaps think of a name like `reana-build-manifests`... Let's muse IRL.